### PR TITLE
Added nginx for docker-compose and config-file creation script

### PR DESCRIPTION
**Pull request by llehtine**

### CODE REVIEW

File: dockerize/create_nginx_conf.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+echo "What is your servers FQDN? "
+read FQDN
+echo "What is your servers IP? "
+read IP_ADDR
+
+cat <<EOF > nginx/b2share.conf
+server {
+        listen 80;
```
Comment (emanueldima) [inline dockerize/create_nginx_conf.sh]: We can not allow unencrypted traffic. Sometimes the web UI sends session confidential information to the server.

Comment (llehtine) [inline dockerize/create_nginx_conf.sh]: Ok. I will have to remove the normal 80-interface from the config.

Comment (llehtine) [inline dockerize/create_nginx_conf.sh]: Although, the OAI-standard does require unencrypted http

Comment (emanueldima) [inline dockerize/create_nginx_conf.sh]: You're right, port 80 should be open for /oai2d only.

**Pull request closed without merging**